jetbrains-core/src/software/aws/toolkits/jetbrains/ui/wizard/PyCharmSdkSelectionPanel.kt

```diff
@@ -78,6 +78,8 @@ class PyCharmSdkSelectionPanel(val step: SamProjectRuntimeSelectionStep) : SdkSe
         sdkSelectionPanel.addChangeListener(Runnable {
             step.checkValid()
         })
+        
```

Comment:
ktlint complains with the error. Can you double check this line to make sure it doesn't have any extra spaces?
```
/codebuild/output/src361109414/src/github.com/aws/aws-toolkit-jetbrains/jetbrains-core/src/software/aws/toolkits/jetbrains/ui/wizard/PyCharmSdkSelectionPanel.kt:81:1: Trailing space(s) (no-trailing-spaces)
```

Reply:
That's so strange, I committed it exactly how it was written in IntelliJ, and had run `gradlew runIde`, which compiled with no errors.

It seems the blankline I left before the extra line is prefixed with 8 spaces for the automatic IDE indentation, whereas other blanklines already in the source file have no indentation.

I'm surprised this is an issue, I've never seen any language or compiler care about something like this before.

Reply:
It's not really the language or compiler, ktlint is a build step that runs to automatically check a source files adherence to our style guidelines.